src/main/java/core/basesyntax/Node.java

```diff
@@ -0,0 +1,47 @@
+package core.basesyntax;
+
+public class Node<K,V> {
+
+    private int hash;
+    private K key;
+    private V value;
+    private Node<K,V> next;
+
+    public Node(K key, V value, Node<K, V> next) {
```

Comment:
This "next" is not needed in the constructor.  The only place this constructor is called has "next" = null
```suggestion
    public Node(K key, V value) {
```